presto-native-execution/presto_cpp/main/LinuxMemoryChecker.h

```diff
@@ -13,12 +13,6 @@
  */
 #pragma once
 
-#include <boost/regex.hpp>
 #include "presto_cpp/main/PeriodicMemoryChecker.h"
 
-namespace facebook::presto {
-
-std::unique_ptr<PeriodicMemoryChecker>& linuxMemoryChecker();
-void createLinuxMemoryChecker(PeriodicMemoryChecker::Config& config);
-
-} // namespace facebook::presto
+namespace facebook::presto {} // namespace facebook::presto
```

Comment:
I believe we don't need this header anymore. The singleton gets created as part of the inclusion in the build on Linux.

Reply:
@czentgr I have deleted this file LinuxMemoryChecker.h since I don't think we need it anymore